### ISSUE TITLE
Children Disappear, Whistle Won't Recall Them

### ISSUE THREAD

**Issue by IAmVensey** [WildBamaBoy/minecraft-comes-alive#1068]
MCA Version: MCA-1.12.x-5.3.1
RadixCore Version: RadixCore-1.12.x-2.2.1
Forge Version: forge-1.12.2-14.23.0.2491

I had over 15 children, and upon opening up my save, some of my oldest children have disappeared. Most of them were my adult male children, though one of my non-adult female children also disappeared.

I have tried using the whistle, but the whistle does not even acknowledge that they exist.

They did not die because I had ordered them all to stay in a well-lit area with a home point, and there were no enemies nearby. I received no death notifications for them. I tried searching any possible places that they might have teleported to, but I still cannot find them.

Is there a cap to the number of children you can have?

**Comment by MentalMouse** [comment]
In the past I have had some trouble with villagers sleepwalking into walls and suffocating., especially at "open corners" (that is, two walls meet to form an interior corner, but where the exterior corner would be are leaves or fences.  (And of course, the generated village has a lot of those corners.)  I'm not sure about death notices, I wasn't noticing any, but chests did appear with their death items.

**Comment by IAmVensey** [comment]
I've seen that too, though it mostly happens with beds in my case.

What's strange is that I had them in 9X9 solid squares (no open corners), and set "Stay" and "Set Home." I did this with every villager, too, not just my children. None of the other villagers went missing, and I didn't see any chests at all in any of the "cubbies" that my children were in, so I assume they didn't die or else their chests should have shown up.

The other thing is that they were there, I saved the game and closed it, and when I reopened it the next day, they were gone. I've tried my backup saves of the world as well but they didn't appear then either.
